darwin/dataset/download_manager.py

```diff
@@ -138,19 +154,31 @@ def download_image_from_json_annotation(
         Path where to download the image
     use_folders: bool
         Recreate folders
+    video_frames: bool
+        Pulls video frames images instead of video files
     """
     annotation = json.load(annotation_path.open())
 
-    # Make the image file name match the one of the JSON annotation
-    original_filename_suffix = Path(annotation["image"]["original_filename"]).suffix
+    filename = annotation_path.stem
+    if video_frames:
```

Comment:
shouldn't we first check if what you are extracting is a video? 

imagine an export that has one video and one image in it.

Reply:
True, I didn't think about that situation. Just added a check.